bgpd/bgp_zebra.c

```diff
@@ -1610,9 +1609,7 @@ bgp_zebra_announce_actual(struct bgp_dest *dest, struct bgp_path_info *info,
 					 table->afi, table->safi, &nhg_id,
 					 &metric, &tag, &allow_recursion);
 
-	is_add = (valid_nh_count || nhg_id) ? true : false;
-
-	if (is_add && CHECK_FLAG(bm->flags, BM_FLAG_SEND_EXTRA_DATA_TO_ZEBRA)) {
+	if (CHECK_FLAG(bm->flags, BM_FLAG_SEND_EXTRA_DATA_TO_ZEBRA)) {
```

Comment:
so on route removal we want to send the extra data?   What is your line of thinking here?

Reply:
ah this is send only right?

Reply:
>ah this is send only right?

yes